server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -363,7 +363,9 @@ private Sink getSink(long timestamp)
                                           }
                                         }
                                     )
-                                )
+                                ),
+                                "query/wait/time",
+                                ImmutableMap.of("segment", theSink.getSegment().getIdentifier())
```

Comment:
This is additional cardinality that adds to the size of metrics emitted.

If my understanding of what this is wrapping is correct this is `query/segment/time` equivalent for realtime nodes, or maybe `query/segmentAndCache/time`

Combining segment id with that metric was explicitly removed due to the large quantity of metrics it created.